modules/csg/csg_shape.cpp

```diff
@@ -741,15 +741,15 @@ CSGBrush *CSGMesh::_build_brush() {
 		const Vector3 *vr = avertices.ptr();
 
 		Vector<Vector3> anormals = arrays[Mesh::ARRAY_NORMAL];
-		const Vector3 *nr;
+		const Vector3 *nr = NULL;
```

Comment:
I don't understand why those trigger a warning though. Their use is fully guarded by the `nr_used` and `uvr_used` booleans. It's not particularly elegant code but I don't see any code path where they'd end up used uninitialized.

Reply:
Yes, the warning is not so smart and I recommend to remove `xyz_used` boolean values and instream use `NULL`.